taworks/taform/templates/taform/application.html

```diff
@@ -1,30 +1,140 @@
+{% load staticfiles %}
+
 {% load static %}
 <script src="{% static 'taform/js/app.js' %}" type="text/javascript" ></script>
-<link rel="stylesheet" type="text/css" href="{% static 'taform/css/style.css' %}" />
 
+<link rel="stylesheet" type="text/css" href="{% static 'taform/css/style.css' %}" />
 
 <body onload="noBack();" onpageshow="if (event.persisted) noBack();" onunload="">
 {% load taform_extras %}
+
+<h1>Management Sciences TA Application Form</h1>
+
 <form enctype="multipart/form-data" autocomplete="off" action="." method="POST" accept-charset="utf-8" >
 {% csrf_token %}
+
+<h3>Basic Information</h3>
+
 <table>
-{{ s_form }}
+<tr>
+<td>
+Student ID:</td>
+	<td> {{s_form.student_id}}</td>
+	<td><font face="Arial", size="2">{{s_form.student_id.help_text}}</font></td>
+</tr>
+<tr>
+	<td>First Name:</td>
+	<td>{{s_form.first_name}}</td>
+	<td><font face="Arial", size="2">{{s_form.first_name.help_text}}</font></td>
+</tr>
+<tr>
+	<td>Last Name:</td>
+	<td>{{s_form.last_name}}</td>
+	<td><font face="Arial", size="2">{{s_form.last_name.help_text}}</font></td>
+</tr>
+<tr>
+	<td>Quest ID:</td>
+	<td>{{s_form.quest_id}}</td>
+	<td><font face="Arial", size="2">{{s_form.quest_id.help_text}}</font></td>
+</tr>
+<tr>
+	<td>Department:</td>
+	<td>{{s_form.department}}</td>
+	<td><font face="Arial", size="2">{{s_form.department.help_text}}</font></td>
+</tr>
+<tr>
+	<td>Current Program:</td>
+	<td>{{s_form.current_program}}</td>
+	<td><font face="Arial", size="2">{{s_form.current_program.help_text}}</font></td>
+</tr>
+<tr>
+	<td>Citizenship:</td>
+	<td>{{s_form.citizenship}}</td>
+	<td><font face="Arial", size="2">{{s_form.citizenship.help_text}}</font></td>
+</tr>
+<tr>
+	<td>Student Visa Expiry Date:</td>
+	<td>{{s_form.student_visa_expiry_date}}</td>
+	<td><font face="Arial", size="2">{{s_form.student_visa_expiry_date.help_text}}</font></td>
+</tr>
+<tr>
+	<td>Enrollment:</td>
+	<td>{{s_form.enrolled_status}}</td>
+	<td><font face="Arial", size="2">{{s_form.enrolled_status.help_text}}</font></td>
+</tr>
+<tr>
+	<td>Completed ExpecTAtions:</td>
+	<td>{{s_form.ta_expectations}}&nbsp;Yes</td>
+	<td><font face="Arial", size="2">{{s_form.ta_expectations.help_text}}</font></td> 
+</tr>
 </table>
+
+<h3>CV Upload</h3>
+<font size="3">To help instructors select TAs, please upload a CV or resume (PDF only):</font><br><br>
+{{s_form.cv}}
+
+<h3>Course Application</h3>
+
+<div id="divWordWrap">
+<b>Application Instructions:</b> You are encouraged to apply for many positions to increase your chance of receiving a TA position. Please verify that you are available for all of the TA requirements by reviewing the schedule for each course before applying. <br><br>
```

Comment:
these long bodies of text needs to be put in paragraph tags and spread across multiple lines.